yarn.lock

```diff
@@ -6616,6 +6616,11 @@ parseurl@~1.3.2, parseurl@~1.3.3:
   resolved "https://registry.yarnpkg.com/parseurl/-/parseurl-1.3.3.tgz#9da19e7bee8d12dff0513ed5b76957793bc2e8d4"
   integrity sha512-CiyeOxFT/JZyN5m0z9PfXw4SCBJ6Sygz1Dpl0wqjlhDEGGBP1GnsUVEL0p63hoG1fcj3fHynXi9NYO4nWOL+qQ==
 
+path-browserify@^1.0.1:
```

Comment:
Tried to get path to work in frontend js, but there ended up being more webpack issues that I could debug. For now, path.join works, but path.relative invokes a "process" package that webpack doesn't resolve correctly. I will need to get a path utility a different way, implement my own, or get some help here